charts/newrelic-infrastructure/values.yaml

```diff
@@ -75,9 +75,23 @@ kubelet:
       effect: "NoExecute"
   nodeSelector: {}
   affinity: {}
+  # -- Config for the Infrastructure agent.
+  # Will be used by the forwarder sidecars and the agent running integrations.
```

Comment:
This one will not be used by the agent running integrations, only by the control plane sidecar.

Reply:
Solved in [#34a64bf](https://github.com/newrelic/nri-kubernetes/pull/422/commits/34a64bfdba0341b3f159726705bcf0e54541c9f4)